[IMP] Domain names: improve current doc (#7073)
odoo/documentation: content/administration/maintain/domain_names.rst

You can use subdomains to direct your visitors to websites other than your main
website or specific pages.

You can use a **custom domain name** to access your Odoo database and websites:

- By :ref:`registering a free domain name with Odoo <domain-name/odoo-register>` (for Odoo Online
  databases);
- By :ref:`configuring a custom domain that you already own <domain-name/existing>`.

.. tip::
   - Odoo offers a :ref:`free custom domain name <domain-name/odoo-register>` to all Odoo Online
     databases for one year.
   - A good domain name is crucial for your brand. To make it effective, keep it simple, short, easy
     to remember, and easy to spell.

.. seealso::
   - `Odoo Tutorials: Register a Free Domain Name
     <https://www.odoo.com/slides/slide/register-a-free-domain-name-1663>`_
   - `Odoo Quick Tips: Get a free domain name! <https://www.youtube.com/watch?v=eAfgeNOHLP4>`_

.. _domain-name/odoo-register:

Register a free domain name with Odoo
=====================================

When creating your site on Odoo, you can enjoy free domain registration for the first year. After
this period, Odoo continues to manage your domain in partnership with **Gandi**, the domain name

> registrar, and you will be charged at the renewal rates as indicated on `their site (renewal column)

Author: I replaced "enjoy" by "get" as discussed

<https://www.gandi.net/en/domain>`_. Odoo then sends a renewal quotation every year to the email
address mentioned as the owner a few weeks before the expiration date of the domain. The domain is
renewed automatically when the quotation is confirmed.

.. note::
   - A one-year free domain name is also available for free Odoo Online databases (if you installed
     one app only, for example). In this case, Odoo reviews your request and website to avoid abuse.

>      This process can take several days due to the success of the offer.

Author: Rephrased: This process can take several days due to the high number of requests.

   manage your domain name.

Manage your mailbox
-------------------

The one-year free domain name offer doesn’t include any mailbox. To link your domain name with your
mailbox, there are different options:

**Option 1**

Create a sub-domain (e.g., `subdomain`.example.com) to use as an alias domain for the database. This allows
you to create records in Odoo from emails received on your @`subdomain`.example.com aliases. To do
so, go to your portal and click the gear icon, :guilabel:`Domain Names`, :guilabel:`DNS`, and click
:guilabel:`ADD DNS RECORD` and choose :guilabel:`CNAME`. Enter your email in the :guilabel:`Host`
field and your dbname.odoo.com. as the :guilabel:`Value`.
Then, add mail.example.com as "own domain" in the database domains.
mail.example.com can then be used as an alias domain rather than dbname.odoo.com.

**Option 2**

You can configure your MX records to use your email server or solution such as Google Workspace.
Create the DNS records required to use your domain name with a third-party email service.
To manage DNS records, go to your Odoo portal, click :guilabel:`My Databases`, then click the gear

> icon, :guilabel:`Domain names`, and :guilabel:`DNS`.

Reviewer: You can configure your MX records to use your email server such as Google Workspace. To do so, create the required DNS records to use your domain name.
To manage DNS records, go to your Odoo portal, click :guilabel:`My Databases`, then click the gear
icon, :guilabel:`Domain names`, and :guilabel:`DNS`.

.. seealso::
  - :doc:`/applications/general/email_communication/email_servers`

.. _domain-name/indexing:

> Indexing of domain names by search engines
> ==========================================

Reviewer 2: ```suggestion
Indexing domain names by search engines
=======================================
```

> Search engines like Google and Bing rely on web crawlers (:dfn:`robots that explore and analyze
> the web`) to discover and index websites and their related domain names. These crawlers find new
> website links through existing pages, enabling search engines to automatically index domain names
> when their URLs are mentioned elsewhere online.

Reviewer 2: ```suggestion
Search engines, like Google and Bing, rely on web crawlers (:dfn:`robots that explore and analyze
the web`) to discover and index websites, and their related domain names. These crawlers find new
website links through existing pages, enabling search engines to automatically index domain names
when their URLs are mentioned elsewhere online.
```

Improving the appearance and positioning of web pages on search engines is a practice called
:ref:`Search Engine Optimization <pages/seo>`.

> Some search engines provide tools for web admins, such as
> `Google Search Console <https://search.google.com/search-console>`_ and
> `Bing Webmaster Tools <https://www.bing.com/webmasters>`_, to help you analyze and improve your
> page ranking. You must prove that you own your domain name to use these services. One way to verify
> the ownership of your domain name is by adding a DNS record (you can do this for :ref:`domain names
> registered with Odoo <domain-name/odoo-manage>` and for domain names managed by other providers) or
> using the :guilabel:`Console Google Search` available in your website's configuration.

Reviewer 2: ```suggestion
Some search engines provide tools for web admins, such as:

`Google Search Console <https://search.google.com/search-console>`_ and `Bing Webmaster Tools
<https://www.bing.com/webmasters>`_, to help users analyze and improve page ranking.

Users **must** prove themselves as the owner of the domain name to use these services.

One way to verify the ownership of the domain name is by adding a DNS record (users can do this for
:ref:`domain names registered with Odoo <domain-name/odoo-manage>` and for domain names managed by
other providers) or by using the :guilabel:`Console Google Search` available in the website's
configuration.
```

.. Important::

>    You cannot remove the subdomain dbname.odoo.com if Odoo hosts you. However, the subdomain
>    dbname.odoo.com exploration by search engines is blocked when you add a custom domain name to the
>    configuration of your website.

Reviewer 2: ```suggestion
   Users **cannot** remove the subdomain dbname.odoo.com if Odoo is the host. However, the subdomain
   dbname.odoo.com exploration by search engines is blocked when users add a custom domain name to
   the configuration of the website.
```

when their URLs are mentioned elsewhere online.

Improving the appearance and positioning of web pages on search engines is a practice called
:ref:`Search Engine Optimization <pages/seo>`.

Some search engines provide tools for web admins, such as
`Google Search Console <https://search.google.com/search-console>`_ and
`Bing Webmaster Tools <https://www.bing.com/webmasters>`_, to help you analyze and improve your
page ranking. You must prove that you own your domain name to use these services. One way to verify
the ownership of your domain name is by adding a DNS record (you can do this for :ref:`domain names
registered with Odoo <domain-name/odoo-manage>` and for domain names managed by other providers) or
using the :guilabel:`Console Google Search` available in your website's configuration.

.. Important::
   You cannot remove the subdomain dbname.odoo.com if Odoo hosts you. However, the subdomain
   dbname.odoo.com exploration by search engines is blocked when you add a custom domain name in the
   configuration of your website.

.. seealso::
   - `Google Search Console Help - Verify your site ownership <https://support.google.com/webmasters/answer/9008080>`_

>    - `Bing Webmaster Tools - Add and Verify site <https://www.bing.com/webmasters/help/add-and-verify-site-12184f8b>`_

Author: you're right!
